apps/wizarr-frontend/src/modules/admin/components/Users/UserList/UserItem.vue

```diff
@@ -66,8 +66,8 @@ export default defineComponent({
     },
     data() {
         return {
-            profilePicture: "https://ui-avatars.com/api/?uppercase=true&name=" + this.user.username + "&length=1",
-            backupPicture: "https://ui-avatars.com/api/?uppercase=true&name=" + this.user.username + "&length=1",
+            profilePicture: "https://ui-avatars.com/api/?uppercase=true&name=" + this.user.username[0],
```

Comment:
What does this do/archive ?

Reply:
> What does this do/archive ?

We're no longer parsing peoples usernames to a third party site. Just stopping unnecessary data leaving the server to a third party when we can just simply parse the first character and achieve the same result.

Reply:
Okay yh thats awesome, fine by me :P